tests/parsers/apache_access.py

```diff
@@ -22,15 +22,15 @@ def testParse(self):
     parser = apache_access.ApacheAccessParser()
     storage_writer = self._ParseFile(['access.log'], parser)
 
-    self.assertEqual(storage_writer.number_of_warnings, 0)
-    self.assertEqual(storage_writer.number_of_events, 5)
+    self.assertEqual(storage_writer.number_of_warnings, 1)
```

Comment:
Please add a test to validate the warning.

Reply:
ack